onnx/onnx.proto

```diff
@@ -247,6 +342,10 @@ message ModelProto {
 
   // Named metadata values; keys should be distinct.
   repeated StringStringEntryProto metadata_props = 14;
+
+  // Training-specific information. Sequentially executing all stored
+  // "TrainingInfoProto"s is one training iteration.
+  repeated TrainingInfoProto training_info = 21;
```

Comment:
What does this mean, given that the initialization is also inside a TrainingInfoProto? I think I understand repeating a list of "update algorithm" in one iteration, but the initialization is a problem.

Reply:
There will be two execution modes.

1. Training: sequentially execute all `TrainingInfoProto`'s `algorithm` graph.
2. Initialization: sequentially execute all `TrainingInfoProto`'s `initialization` graph.